gears/src/application/handlers.rs

```diff
@@ -37,15 +46,7 @@ pub trait TxHandler {
         chain_id: tendermint::informal::chain::Id,
         fee: Option<SendCoins>,
     ) -> anyhow::Result<Response> {
-        let fee = Fee {
-            amount: fee,
-            gas_limit: 100000000, //TODO: remove hard coded gas limit
```

Comment:
Some of theses values need to be added as CLI params - that's what the TODO is alluding to.

Reply:
You're right. 
It's easy way to construct default instance and reuse it anywhere. 

In Rust world, the builder pattern is widely used. I like it for such cases when we can setup field by field. But default constructors are much simpler.